genesis-tools/contracts/lockup-contract/src/lib.rs

```diff
@@ -0,0 +1,96 @@
+//! A smart contract that allows tokens lockup.
+
+use borsh::{BorshDeserialize, BorshSerialize};
+use near_bindgen::{env, near_bindgen, Promise};
+
+#[global_allocator]
+static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
+
+/// A key is a sequence of bytes, potentially including the prefix determining the cryptographic type
+/// of the key. For forward compatibility we do not enforce any specific length.
+type PublicKey = Vec<u8>;
+
+#[near_bindgen]
+#[derive(BorshDeserialize, BorshSerialize)]
+pub struct LockupContract {
+    /// The amount in yacto-NEAR tokens locked for this account.
```

Comment:
It's `yocto` https://en.wikipedia.org/wiki/Yocto- or just `yN`

also `to_yocto` seems like a very useful helper for bindgen.